[ES] Update `Client/Beatmap editor/AiMod` (#10535)
ppy/osu-wiki: wiki/Client/Beatmap_editor/AiMod/es.md

| This hold note is less than 10ms long! | Se presenta una nota mantenida con una longitud excesivamente corta, lo que hace que el mapa sea imposible para conseguir una SS. | Aumenta la duración de la nota mantenida o elimínala. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
| This object overlaps with another object. | Dos objetos se superponen entre sí en la misma columna en el tiempo, lo que hace que el mapa sea imposible para conseguir una SS. | Retira uno de los objetos superpuestos. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
| This object is stacked on top of another object. | Dos objetos se apilan uno encima del otro en la misma columna en el tiempo, lo que hace que el mapa sea imposible para conseguir una SS. | Encuentra las teclas apiladas o notas mantenidas y mueve o elimina una de ellas. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |

> | More than 6 notes simultaneously is not allowed. | Uno o más patrones en el mapa requieren que el jugador presione activamente más de 6 teclas a la vez. En ciertos casos, los [criterios de clasificación de osu!mania](/wiki/Ranking_criteria/osu!mania) lo desaconsejan, ya que la mayoría de los teclados solo permiten hasta 6 pulsaciones de teclas a la vez debido a un fenómeno conocido como [ghosting](https://nelson-miller.com/what-is-keyboard-ghosting-and-how-do-you-prevent-it/). | Revisa tu mapa para asegurarte de que cada columna tenga como máximo 6 pulsaciones de notas simultáneas. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |

Reviewer: ```suggestion
| More than 6 notes simultaneously is not allowed. | Uno o más patrones del mapa requieren que el jugador pulse activamente más de 6 teclas a la vez. En ciertos casos, los [criterios de clasificación de osu!mania](/wiki/Ranking_criteria/osu!mania) lo desaconsejan, ya que la mayoría de los teclados solo permiten pulsar hasta 6 teclas a la vez debido a un fenómeno conocido como [ghosting](https://nelson-miller.com/what-is-keyboard-ghosting-and-how-do-you-prevent-it/). | Revisa tu beatmap para asegurarte de que cada columna tiene como máximo 6 pulsaciones de nota simultáneas. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
```

#### Design

| Mensaje | Explicación | Solución | Notas |
| :-- | :-- | :-- | :-- |
| Your beatmap has no background image. | Tu beatmap no tiene imagen de fondo. | Encuentra una imagen adecuada y úsala como imagen de fondo para la dificultad. | Los vídeos no cuentan como imágenes de fondo porque el usuario no está obligado a descargarlos. |

> | File missing:{0} | A tu mapa le faltan ciertos archivos. | Recupera los archivos que faltan o asegúrate de que el beatmap no esté tratando de usar archivos que no existen. |  |

Reviewer: ```suggestion
| File missing:{0} | A tu beatmap le faltan ciertos archivos. | Recupera los archivos que faltan o asegúrate de que el beatmap no esté tratando de usar archivos que no existen. |  |
```

#### Timing

| Mensaje | Explicación | Solución | Notas |
| :-- | :-- | :-- | :-- |

> | All timing sections have a volume below 5%. | Todas las secciones de temporización tienen un volumen inferior al 5 %. | Configura al menos una sección de temporización para que tenga un volumen de sonido de 5 % o más. |  |

Reviewer: ```suggestion
| All timing sections have a volume below 5%. | Todas las secciones de tiempo tienen un volumen inferior al 5 %. | Configura al menos una sección de tiempo para que tenga un volumen de sonido de 5 % o más. |  |
```

#### Meta

| Mensaje | Explicación | Solución | Notas |
| :-- | :-- | :-- | :-- |

> | Drain time should be over 30 seconds. | La duración de tu mapa de principio a fin, sin incluir los tiempos de descanso, es inferior a 30 segundos. | Intenta usar un spinner para terminar el mapa si tu canción lo admite. De lo contrario, tu canción puede ser demasiado corta. En tal caso, intenta extender tu archivo .mp3 para alcanzar la marca de 30 segundos. | AiMod no detecta spinners que pueden extenderse más allá de la marca de tiempo de drenaje de 30 segundos. |

Reviewer: ```suggestion
| Drain time should be over 30 seconds. | La duración de tu mapa de principio a fin, sin incluir los descansos, es inferior a 30 segundos. | Intenta usar un spinner para terminar el mapa si tu canción lo admite. De lo contrario, tu canción puede ser demasiado corta. En tal caso, intenta extender tu archivo .mp3 para alcanzar la marca de 30 segundos. | AiMod no detecta los spinners que pueden sobrepasar los 30 segundos de drenaje. |
```

| {0}'s dimensions must be {1}x{1} | La dimensión del elemento de diseño especificado no coincide con su valor predeterminado previsto. | Cambia el tamaño del elemento de diseño en cuestión a su resolución adecuada. |  |
| Your video's dimensions must not exceed 1024x768 for the 4:3 format. | Las dimensiones de tu vídeo no deben superar 1024x768 para el formato 4:3. | Cambia el tamaño o reemplaza el vídeo de fondo. |  |
| Your video's dimensions must not exceed 1280x720 for the 16:9 format. | Las dimensiones de tu vídeo no deben superar 1280x720 para el formato 16:9. | Cambia el tamaño o reemplaza el vídeo de fondo. |  |

#### Timing

| Mensaje | Explicación | Solución | Notas |
| :-- | :-- | :-- | :-- |
| This beatmap is over 6 minutes long. Consider shortening it if it's not a marathon-style map. | Este beatmap dura más de 6 minutos. Considera acortarlo si no es un mapa estilo maratón. | Agrega más descansos en el beatmap o corta la canción. | Esta advertencia se puede ignorar con seguridad si la longitud del mapa en cuestión es la prevista. |

> | Your beatmap is shorter than 45 seconds. Consider making it longer. | Tu beatmap tiene menos de 45 segundos. Considera hacerlo más largo. | Mapea más partes de la canción. | Esta advertencia se puede ignorar con seguridad si la longitud del mapa en cuestión es la prevista. |

Reviewer: ```suggestion
| Your beatmap is shorter than 45 seconds. Consider making it longer. | Tu beatmap dura menos de 45 segundos. Considera hacerlo más largo. | Mapea más partes de la canción. | Esta advertencia se puede ignorar con seguridad si la longitud del mapa en cuestión es la prevista. |
```

> | Audio bitrate is higher than 192 kbps. Consider recompressing to CBR 192 kbps or VBR ~1.0. | El archivo `.mp3` del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) superior a 192 kbps, que supera la asignación de los [criterios de clasificación](/wiki/Ranking_criteria). | Vuelve a codificar el archivo `.mp3` para que su tasa de bits esté entre 128 kbps y 192 kbps (siendo 192 kbps la opción recomendada). | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |
> | Audio bitrate is lower than 128 kbps. Consider finding a better quality source. | El archivo .mp3 del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) inferior a 128 kbps, que está por debajo de la asignación de los [criterios de clasificación](/wiki/Ranking_criteria). | Cambia el archivo .mp3 a uno con una tasa de bits más alta. | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |

Reviewer: ```suggestion
| Audio bitrate is higher than 192 kbps. Consider recompressing to CBR 192 kbps or VBR ~1.0. | El archivo `.mp3` del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) superior a 192 kbps, que supera lo permitido en los [criterios de clasificación](/wiki/Ranking_criteria). | Vuelve a codificar el archivo `.mp3` para que su tasa de bits esté entre 128 kbps y 192 kbps (siendo 192 kbps la opción recomendada). | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |
| Audio bitrate is lower than 128 kbps. Consider finding a better quality source. | El archivo .mp3 del beatmap está codificado en una [tasa de bits](https://es.wikipedia.org/wiki/Tasa_de_bits) inferior a 128 kbps, que está por debajo de lo permitido en los [criterios de clasificación](/wiki/Ranking_criteria). | Cambia el archivo .mp3 a uno con una tasa de bits más alta. | Aunque esta advertencia existe en el juego, AiMod nunca muestra la advertencia debido a limitaciones técnicas. |
```

| Kiai time is toggled on for less than 15 seconds. | El Kiai time se activa durante menos de 15 segundos. | Extiende la duración del kiai time a más de 15 segundos. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |

> | The mp3 you are using is a lot longer than the part which is mapped. Consider cropping it to reduce filesize. | El mp3 que estás utilizando es mucho más largo que la parte mapeada. Considera recortarlo para reducir el tamaño del archivo. | Extiende el beatmap o corta la canción. |  |

Reviewer: ```suggestion
| The mp3 you are using is a lot longer than the part which is mapped. Consider cropping it to reduce filesize. | El mp3 que estás usando es mucho más largo que la parte mapeada. Considera recortarlo para reducir el tamaño del archivo. | Extiende el beatmap o corta la canción. |  |
```

| Kiai needs an end time point. | El kiai necesita un punto de finalización. | Agrega un nuevo punto de tiempo heredado para que sirva como punto final del kiai. |  |
| A preview point for this map is not set. Consider setting one from the Timing menu. | No se ha establecido un punto de vista previa para este mapa. Considera configurar uno desde el menú Timing. | Establece un punto de vista previa para el mapa haciendo clic en `Timing` > `Seleccionar la posición actual como punto de vista previa`. |  |
| Two timing points exist at the same time! | ¡Existen dos puntos de tiempo al mismo tiempo! | Elimina una de las secciones de temporización en conflicto. |  |

> | {0} out of {1} timing sections have a volume below 5%. | {0} de {1} secciones de temporización tienen un volumen inferior al 5 %. | Establece el volumen de los hitsounds de las secciones de temporización silenciosas en un 5 % o más. |  |

Reviewer: ```suggestion
| {0} out of {1} timing sections have a volume below 5%. | {0} de {1} secciones de tiempo tienen un volumen inferior al 5 %. | Establece el volumen de los hitsounds de las secciones de tiempo silenciosas en un 5 % o más. |  |
```

> | More than 1/3 of the map is Kiai time. Consider reducing this. | Más de 1/3 del mapa es Kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
> | More than 1/2 of the TV Size map is Kiai time. Consider reducing this. | Más de la mitad del mapa TV Size es Kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |

Reviewer: ```suggestion
| More than 1/3 of the map is Kiai time. Consider reducing this. | Más de 1/3 del mapa es kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
| More than 1/2 of the TV Size map is Kiai time. Consider reducing this. | Más de la mitad del mapa TV Size es kiai time. Considera reducir esto. | Reduce el uso del kiai en todo el mapa. | Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
```

| Kiai isn't snapped! | El punto de partida del kiai especificado no se ajusta a la línea de tiempo. | Ajusta correctamente el punto de inicio del kiai a su posición deseada en la línea de tiempo. |  |

> | Kiai's end isn't snapped! | El punto final del kiai especificado no se ajusta a la línea de tiempo. | Ajusta correctamente el punto final del kiai a su posición deseada en la línea de tiempo. |  |

Reviewer: ```suggestion
| Kiai's end isn't snapped! | El punto de finalización del kiai especificado no se ajusta a la línea de tiempo. | Ajusta correctamente el punto de finalización del kiai a su posición deseada en la línea de tiempo. |  |
```

> | Breaktime is not suggested for mania maps. | No se sugiere tiempo de descanso para los mapas de mania. | Mapea la parte donde ocurre el tiempo de descanso. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania. Esta advertencia se puede ignorar con seguridad si se pretende utilizar el tiempo de descanso en cuestión, o si la sección de la canción no se puede mapear. |

Reviewer: ```suggestion
| Breaktime is not suggested for mania maps. | No se sugiere tiempo de descanso para los mapas de mania. | Mapea la parte donde ocurre el tiempo de descanso. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania. Esta advertencia se puede ignorar con seguridad si se pretende usar el tiempo de descanso en cuestión, o si la sección de la canción no se puede mapear. |
```

> | <!-- Can't confirm -->Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad de los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad de los sliders en la dificultad Easy/Normal en cuestión. |  |
> | <!-- Can't confirm -->Kiai is toggled very frequently! | ¡El kiai se activa con mucha frecuencia! | Reduce la cantidad de ocurrencias del kiai en todo el mapa. | También conocidos como *kiai flashes*. Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |

Reviewer: ```suggestion
| Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad de los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad de los sliders en la dificultad Easy/Normal en cuestión. |  |
| Kiai is toggled very frequently! | ¡El kiai se activa con mucha frecuencia! | Reduce la cantidad de ocurrencias del kiai en todo el mapa. | También conocidos como *kiai flashes*. Esta advertencia se puede ignorar con seguridad si se pretende el uso del kiai en cuestión. |
```

> | Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad de los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad de los sliders en la dificultad Easy/Normal en cuestión. |  |

Reviewer: ```suggestion
| Easy/Normal diff contains too many speed changes. | La dificultad Easy/Normal presenta una cantidad frecuente de cambios de velocidad en los sliders, lo cual no se recomienda en los [criterios de clasificación](/wiki/Ranking_criteria). | Reduce el número de cambios de velocidad en los sliders en la dificultad Easy/Normal en cuestión. |  |
```

| Romanised artist contains unicode. | El campo `Romanised Artist` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) contiene [caracteres Unicode no estándar](https://en.wikipedia.org/wiki/List_of_Unicode_characters). | Romaniza el nombre del artista en el campo `Romanised Artist` siguiendo las [reglas de estandarización de metadatos de los criterios de clasificación](/wiki/Ranking_criteria#metadatos). |  |
| Romanised title contains unicode. | El campo `Romanised Title` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) contiene [caracteres Unicode no estándar](https://en.wikipedia.org/wiki/List_of_Unicode_characters). | Romaniza el nombre del título de la canción en el campo `Romanised Title` siguiendo las [reglas de estandarización de metadatos de los criterios de clasificación](/wiki/Ranking_criteria#metadatos). |  |
| Countdown is not allowed in mania mode. | La cuenta regresiva no está permitida en el modo mania. | Desmarca la opción `Enable countdown` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup). | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |

> | Letterboxing is not allowed in mania mode. | El Letterboxing no está permitido en el modo mania. | Desmarca la opción `Letterbox during breaks` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup). | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |

Reviewer: ```suggestion
| Letterboxing is not allowed in mania mode. | El letterboxing no está permitido en el modo mania. | Desmarca la opción `Letterbox during breaks` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup). | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania |
```

| Title conflicts with {0} diff. | El campo `Romanised Title` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que el campo `Romanised Title` sea consistente en todas las dificultades. |  |
| Source conflicts with {0} diff. | El campo `Source` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que el campo `Source` sea consistente en todas las dificultades. |  |
| Tags conflict with {0} diff. | El campo `Tags` en la pestaña General de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que el campo `Tags` sea consistente en todas las dificultades. |  |
| Uninherited timing points conflict with {0} diff. | La posición de los puntos de tiempo no heredados ([líneas rojas](/wiki/Client/Beatmap_editor/Timing#puntos-de-tiempo-no-heredados)) difiere entre una dificultad y otra. | Asegúrate de que la posición de todos los puntos de tiempo no heredados sean consistentes en todas las dificultades. |  |

> | Audio file conflicts with {0} diff. | El archivo .mp3 utilizado en el mapa difiere entre una dificultad y otra. | Asegúrate de que todas las dificultades utilicen el mismo archivo .mp3 como base. |  |

Reviewer: ```suggestion
| Audio file conflicts with {0} diff. | El archivo .mp3 usado en el mapa difiere entre una dificultad y otra. | Asegúrate de que todas las dificultades usen el mismo archivo .mp3 como base. |  |
```

| Countdown conflicts with {0} diff. | La opción `Enable countdown` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que la opción `Enable countdown` esté establecida con la misma configuración en todas las dificultades. |  |
| Letterbox in breaks conflicts with {0} diff. | La opción `Letterbox during breaks` en la pestaña Design de [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) difiere entre una dificultad y otra. | Asegúrate de que la opción `Letterbox during breaks` esté establecida con la misma configuración en todas las dificultades. |  |
| Audio preview time conflicts with {0} diff. | El punto de vista previa del audio difiere entre una dificultad y otra. | Establece manualmente el punto de vista previa de cada dificultad haciendo clic en `Timing` > `Seleccionar la posición actual como punto de vista previa` para que todos aterricen en el mismo lugar en todas las dificultades. |  |

> | Beatmap exceeds allowed upload size ({0}kb allowed) | El archivo `.osz` comprimido es demasiado grande. | Elimina los archivos no utilizados de la carpeta del mapa y comprime los activos (imágenes, vídeos, hitsounds, etc.) que tengan un tamaño inflado. | Consulta [este artículo](/wiki/Beatmapping/Beatmap_submission#limitaciones) para obtener más información sobre las limitaciones de envío de osu!. |

Reviewer: ```suggestion
| Beatmap exceeds allowed upload size ({0}kb allowed) | El archivo `.osz` comprimido es demasiado grande. | Elimina los archivos no usados de la carpeta del mapa y comprime los recursos (imágenes, vídeos, hitsounds, etc.) que tengan un tamaño inflado. | Consulta [este artículo](/wiki/Beatmapping/Beatmap_submission#limitaciones) para obtener más información sobre las limitaciones de envío de osu!. |
```

| This mapset needs at least 2 osu!catch diffs. | Este conjunto de mapas necesita al menos 2 dificultades de osu!catch. | Crea una segunda dificultad para el conjunto. | ![](/wiki/shared/mode/catch.png) solo mapas de osu!catch. Esta advertencia se basa en una regla de los criterios de clasificación arcaica que desde entonces se ha vuelto obsoleta. Por lo tanto, se puede ignorar con seguridad siempre que el mapa siga el [requisito de distribución de los criterios de clasificación actuales](/wiki/Ranking_criteria/osu!mania#reglas). |
| This mapset needs at least 2 osu!mania diffs. | Este conjunto de mapas necesita al menos 2 dificultades de osu!mania. | Crea una segunda dificultad para el conjunto. | ![](/wiki/shared/mode/mania.png) solo mapas de osu!mania. Esta advertencia se basa en una regla de los criterios de clasificación arcaica que desde entonces se ha vuelto obsoleta. Por lo tanto, se puede ignorar con seguridad siempre que el mapa siga el [requisito de distribución de los criterios de clasificación actuales](/wiki/Ranking_criteria#beatmap). |

## Defectos

> - **AiMod no puede detectar una gran cantidad de problemas** que se consideran imposibles de clasificar según los [criterios de clasificación](/wiki/Ranking_criteria), como objetos potencialmente mal encajados, hitsounds retrasados, imagen de fondo superior a 2.5 MB, etc.

Reviewer: ```suggestion
- **AiMod no puede detectar una gran cantidad de problemas** que se consideran imposibles de clasificar según los [criterios de clasificación](/wiki/Ranking_criteria), como objetos potencialmente mal encajados, hitsounds retrasados, imagen de fondo superior a 2,5 MB, etc.
```

> - **AiMod puede advertirle de problemas que no son**, como que el conjunto de mapas necesita dos dificultades estándar en un mapa de [maratón](/wiki/Beatmap/Marathon) y que el [kiai time](/wiki/Gameplay/Kiai_time) se activa por menos de 15 segundos.

Reviewer: ```suggestion
- **AiMod puede advertirte de problemas que no son**, como que el conjunto de mapas necesita dos dificultades estándar en un mapa de [maratón](/wiki/Beatmap/Marathon) y que el [kiai time](/wiki/Gameplay/Kiai_time) se activa por menos de 15 segundos.
```

- **Si `Check distance snap` está activo, AiMod no revisará la distancia entre combos.**
- **Si hay múltiples archivos MP3 en la carpeta del beatmap, AiMod puede advertirte sobre múltiples archivos de canciones**, sin embargo, los arvchivos MP3 también pueden ser usados para efectos de sonido como los aplausos.

> Es recomendable usar otros programas como [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier/releases) hecho por ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) en conjunto con AiMod si lo que buscas es hacer que tu beatmap esté [clasificado](/wiki/Beatmap/Category#ranked). Otros programas están más actualizados con los criterios de clasificación y pueden detectar muchos más problemas que AiMod. No obstante, no ignores completamente a AiMod, ya que algunas reglas en los criterios de clasificación las evalúa dicho programa.

Reviewer: ```suggestion
Es recomendable usar otros programas como [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier/releases) por ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) en conjunto con AiMod si lo que buscas es [clasificar](/wiki/Beatmap/Category#ranked) tu beatmap. Otros programas están más actualizados con los criterios de clasificación y pueden detectar muchos más problemas que AiMod. No obstante, no ignores completamente a AiMod, ya que algunas reglas en los criterios de clasificación las evalúa dicho programa.
```
